linkerd/proxy/client-policy/src/lib.rs

```diff
@@ -590,4 +604,56 @@ pub mod proto {
             }
         }
     }
+
+    impl TryFrom<outbound::FailureAccrual> for FailureAccrual {
+        type Error = InvalidFailureAccrual;
+        fn try_from(accrual: outbound::FailureAccrual) -> Result<Self, Self::Error> {
+            use outbound::failure_accrual::{self, ConsecutiveFailures};
+            let kind = accrual.kind.ok_or(InvalidFailureAccrual::Missing("kind"))?;
+            match kind {
+                failure_accrual::Kind::ConsecutiveFailures(ConsecutiveFailures {
+                    max_failures,
+                    backoff,
+                }) => {
+                    // TODO(eliza): if other failure accrual kinds are added
+                    // that also use exponential backoffs, this could be factored out...
+                    let outbound::ExponentialBackoff { min_backoff, max_backoff, jitter_ratio } =
+                        // TODO(eliza): should this be defaulted?
```

Comment:
i think it's the controller's responsibility to provide a default, if it wants us to use failure accrual...
```suggestion
```